clang/test/CodeGen/target-features-error-2.c

```diff
@@ -15,7 +15,7 @@ int baz(__m256i a) {
 
 #if NEED_AVX_2
 __m128 need_avx(__m128 a, __m128 b) {
-  return _mm_cmp_ps(a, b, 0); // expected-error {{'__builtin_ia32_cmpps' needs target feature avx}}
+  return _mm_cmp_ps(a, b, 8); // expected-error {{'__builtin_ia32_cmpps' needs target feature avx}}
```

Comment:
@phoebewang @RKSimon @KanRobert 
This change shows the behavior of compiling these intrinsics with parameter of [8,31] after this PR.